label_studio/ml_model_providers/models.py

```diff
@@ -0,0 +1,58 @@
+"""This file and its contents are licensed under the Apache License 2.0. Please see the included NOTICE for copyright information and LICENSE for a copy of the license.
+"""
+
+from django.conf import settings
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class ModelProviderConnection(models.Model):
+
+    class ModelProviders(models.TextChoices):
+        OPENAI = 'OpenAI', _('OpenAI')
+
+
+    class ModelProviderConnectionScopes(models.TextChoices):
+        ORG = 'Org', _('Organization')
```

Comment:
nit: it's slightly confusing having the display name be different from the name in the enum. Could we go with
```suggestion
        ORGANIZATION = 'Organization', _('Organization')
```
?

Reply:
Yeah I thought about this, for some reason in all of their docs / examples the value being stored is a short version of the display name - but I agree it will be simpler if they are the same I will go with that